src/bhost_fpga.py

```diff
@@ -22,19 +22,10 @@
 
 class FpgaBHost(FpgaHost):
     def __init__(self, host, katcp_port=7147, boffile=None,
-                 connect=True, config=None,
-                 simulate=False, optimisations=True):
+                 connect=True, config=None):
         FpgaHost.__init__(self, host, katcp_port=katcp_port, boffile=boffile, connect=connect)
         self.config = config
-        # if self.config is not None:
-        #     self.bf_per_fpga = int(self.config['bf_be_per_fpga'])
         self.bf_per_fpga = 4
-        #  optimisations on writes (that currently expose bugs)
-        self.optimisations = optimisations
-        #  simulate operations (and output debug data).
-        # Useful for when there is no hardware
-        self.simulate = simulate
-
 
         # self.spead_initialise()
         LOGGER.info('Beamformer created')
```

Comment:
Btw do we need to specify if the register is read using blindwrite? I kept this because it's in the original code.